src/jobflow_remote/remote/host/remote.py

```diff
@@ -354,20 +351,16 @@ def inter_handler(title, instructions, prompt_list):
     Handler function for interactive prompts from the server.
     Used by Interactive AuthSource.
     """
-
     if title:
-        print(title.strip())
+        pass
```

Comment:
Does ruff have a rule to remove print? I don't think these are load-bearing but if removed they should be replaced with appropriate debugging aids/logging (we don't always have a debugger available when running this code)

Reply:
`ruff` does have such a rule: `T201`. should have ignored that rule